src/components/ConfDeletePopup.js

```diff
@@ -0,0 +1,31 @@
+import Popup from "./Popup";
+
+class ConfirmDeletePopup extends Popup {
+  constructor({ popupSelector, handleFormSubmit }) {
+    super(popupSelector);
+    this._popupForm = this.popupElement.querySelector(".popup__form");
+    this._handleFormSubmit = handleFormSubmit;
+    this._setEventListeners();
+  }
+
+  // Event Listeners setter and submit logics
+  _setEventListeners() {
+    this._popupForm.addEventListener("submit", (evt) => {
+      evt.preventDefault();
+      if (
+        this._handleFormSubmit &&
+        typeof this._handleFormSubmit === "function"
+      ) {
+        this._handleFormSubmit();
+        this.close();
+      }
+    });
+    super.setEventListeners();
+  }
+
+  open() {
+    super.open();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.